udfs/migration/teradata/otranslate.sql

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE OR REPLACE FUNCTION td.otranslate(source_string STRING, from_string STRING, to_string STRING) AS (
```

Comment:
Looks like the otranslate and translate are the same. 

Can we just call translate in otranslate?

Reply:
OTRANSLATE is for teradata and TRANSLATE is for redshift.

Reply:
Can we refer one to the other, so that we can have to keep them in sync if any future change is needed?

Reply:
I can create `translate` function in `community` and call it from `rs.translate` and `td.translate`.

Reply:
sgtm